File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Models/Dtos/PacHeadDto.cs
using System;

namespace Cen.Wms.Client.Models.Dtos
{
    public class PacHeadDto : PacKeyDto
    {
        public string PacCode { get; set; }

        public DateTime DeliveryDate { get; set; }
        public string SupplierName { get; set; }
        public string Gate { get; set; }

        public decimal TotalLinesSum { get; set; }
        public long TotalLinesCount { get; set; }
    }
}

File: TCDNew-master/src/ui.android/lib/models/dtos/PacHeadDto.dart
import 'PacKeyDto.dart';

class PacHeadDto extends PacKeyDto {
  String pacCode;
  DateTime deliveryDate;
  String supplierName;
  String gate;

  num totalLinesSum;
  int totalLinesCount;

  PacHeadDto(purchaseId, this.pacCode, this.deliveryDate, this.supplierName, this.gate, this.totalLinesSum, this.totalLinesCount)
      : super(pacId: purchaseId);

  factory PacHeadDto.fromJson(Map<String, dynamic> json) {
    PacHeadDto data;

    if (json != null) {
      data = PacHeadDto(
        json['pacId'],
        json['pacCode'],
        DateTime.fromMillisecondsSinceEpoch(json['deliveryDate'], isUtc: true),
        json['supplierName'],
        json['gate'],
        json['totalLinesSum'],
        json['totalLinesCount'],
      );
    }
    return data;
  }
}
